### PR TITLE
Pratiksha work

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

**Review by VIP5614 — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

**Review by sanketmod — APPROVED** [review]

File: Pratiksha/Javascript/Assignment7/Q2.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      document.write("<br> After  Sorting ");
+      for (var i = 0; i < stdCount; i++) {
+        document.write(
+          "<br>" + " input_arr[" + i + "]:" + input_arr[i] + "<br>"
+        );
+      }
+    </script>
+  </body>
+</html>
+
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment7/Q2.html:45]: This progam is  not giving the desired output, please check it.

File: Pratiksha/Javascript/Assignment7/Q4.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <title>Selection Sort</title>
+  </head>
+  <body>
+    <script>
+      //Defining array
+      var stdCount = parseInt(prompt("how many student u want to enter?"));
+      var input_arr = [];
+      //Input
+      for (var i = 0; i < stdCount; i++) {
+        for (var mark = 0; mark < 5; mark++) {
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment7/Q4.html:16]: why there is a need of for within a for, if i enter number of students,it is reading marks for 35 students

File: Pratiksha/Javascript/Assignment7/Q5.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Insetion_Sort</title>
+  </head>
+  <body>
+    <script>
+      //Defining array
+      var stdCount = parseInt(prompt("how many student u want to enter?"));
+      var input_arr = [];
+      //Input
+      for (var i = 0; i < stdCount; i++) {
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment7/Q5.html:15]: need to correct

File: Pratiksha/Javascript/Assignment9/Q1.html [hunk 4]
```diff
@@ -1,25 +1,22 @@
...
-        }
+      var a = prompt(" enter a no");
+      console.log(a);
+      var b = a.split("").reverse().join("");
+      console.log(b);
+      if (a == b) {
+        document.write("Palindrome");
+      } else {
+        document.write("not palindrome");
+      }
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment9/Q1.html:19]: for the number u need to use the / and % logic

File: Pratiksha/Javascript/Assignment9/Q3.html [hunk 5]
```diff
@@ -4,25 +4,24 @@
...
+          }
         }
+        return temp;
+      }
     </script>
   </body>
-</html>
\ No newline at end of file
+</html>
+
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment9/Q3.html:27]: program is not giving me the correct ouput. Please debug the program

File: Pratiksha/Javascript/Assignment9/Q4.html [hunk 6]
```diff
@@ -4,19 +4,17 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Q4</title>
+    <title>String_palindrome</title>
   </head>
   <body>
     <script>
       const string = prompt("Enter a string: ");
       const value = checkPalindrome(string);
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment9/Q4.html:12]: this value is never printed in the program, so i don't get the output

File: Pratiksha/Javascript/Assignment9/Q5.html [hunk 7]
```diff
@@ -1,25 +1,24 @@
...
+      let counter = 0;
+      for (let i = 0; i < str.length; i++) {
+        if (str.charAt(i) == "") {
+          counter++;
+        }
+      }
+      console.log(counter);
+    </script>
+  </body>
+</html>
```
Comment (Snehal-113) [inline Pratiksha/Javascript/Assignment9/Q5.html:23]: not getting the correct output.Also the output is displayed on console and not on web page

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**